zio-http/jvm/src/main/scala/zio/http/netty/server/ServerInboundHandler.scala

```diff
@@ -92,7 +92,8 @@ private[zio] final case class ServerInboundHandler(
             if (attemptImmediateWrite(ctx, req.method, exit)) {
               releaseRequest()
             } else {
-              writeResponse(ctx, runtime, exit, req)(releaseRequest)
+              val scope = Scope.unsafe.make
+              writeResponse(ctx, runtime, scope.use(exit), req)(releaseRequest)
```

Comment:
Can this not close the scope before the response is written? So, for example, if a resource in the Scope is a scoped stream, it can close before being fully written to the response by being closed here when the handler finishes, and not when actually writing it is done

Reply:
You are right, it needed to be one level more down. I added a test for streaming, that failed with this code and is now green after pulling down the scope one level. Thanks for the review.